corehq/blobs/s3db.py

```diff
@@ -76,10 +86,10 @@ def put(self, content, **blob_meta_args):
                 s3_bucket.upload_fileobj(content, meta.key)
         return meta
 
-    def get(self, key):
+    def get(self, key, bucket=None):
```

Comment:
Sad to see this come back. [I just removed it](https://github.com/dimagi/commcare-hq/pull/22634/commits/725265bcc8829a9e010f0115033b3c5e1eeeb6f4#diff-1d9ade19245ac6fef5d1bc183aee240aL68), and it made the API much nicer.